packet/flatcar-linux/kubernetes/require.tf

```diff
@@ -2,32 +2,14 @@
 
 terraform {
   required_version = ">= 0.12.0"
-}
-
-provider "ct" {
-  version = "0.4.0"
-}
-
-provider "local" {
-  version = "~> 1.2"
-}
-
-provider "null" {
-  version = "~> 2.1"
-}
-
-provider "template" {
-  version = "~> 2.1"
-}
-
-provider "tls" {
-  version = "~> 2.0"
-}
-
-provider "packet" {
-  version = "~> 2.7.3"
-}
 
-provider "aws" {
-  version = "2.31.0"
+  required_providers {
+    aws      = "2.31.0"
```

Comment:
I the think previous (deleted) missing `~>` was probably a bug introduced recently (tf 0.12 migration?)

Reply:
Could be, but I'd say it's out of scope for this PR to fix that?

Reply:
Created #143 for it.

Reply:
:ok_hand:

Reply:
> I the think previous (deleted) missing `~>` was probably a bug introduced recently (tf 0.12 migration?)

It wasn't me :stuck_out_tongue_closed_eyes: